csrc/ops/alias.cpp

```diff
@@ -77,21 +77,21 @@ TensorView* tryStaticReshape(
     const std::vector<Val*>& new_sizes) {
   std::vector<int64_t> inp_sizes(inp_dom.size());
   for (const auto i : c10::irange(inp_dom.size())) {
-    auto id = inp_dom.at(i);
-    auto id_size = id->extent()->getInt();
-    if (!id_size.has_value()) {
+    IterDomain* id = inp_dom[i];
+    Val* id_size = id->getMaybeExpandedExtent();
```

Comment:
Looks like this is the only logic change in this PR?

Reply:
No. Also the use of `evaluateInt`. For expanded broadcast dimensions, there's an extra indirection that `getInt` or `value` doesn't just work.

Reply:
Got ya. Thanks for explaining.